queries.sql

```diff
@@ -1,3 +1,8 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```

Comment:
- [ ] You must have wanted to write 2020-01-01 because it is >= the date specified, also the start date should be 2015-12-31.